libs/server/src/lib/utils/utils.ts

```diff
@@ -290,3 +290,38 @@ export function getPrimitiveValue(value: any): string | undefined {
     return undefined;
   }
 }
+
+function renameProperty(obj: any, from: string, to: string) {
```

Comment:
We do it in this way in the Nx repo itself cause we need to preserve the order of properties (cause we are serializing the json back to the file). You don't do the serialization, so the order should not matter. So you can do the rename as follows:

```
obj[to] = obj[from]
delete obj[from]
```